nanoFramework.Benchmark/Result/MethodResult.cs

```diff
@@ -1,12 +1,8 @@
-﻿using nanoFramework.Benchmark.Helpers;
-using nanoFramework.Benchmark.Result.Attributes;
-using System;
-using System.Collections;
-using System.Reflection;
+﻿using System;
 
 namespace nanoFramework.Benchmark.Result
 {
-    public class MethodResult
+    class MethodResult
```

Comment:
Visibility is always required! Guess this will do the trick:
```suggestion
    private class MethodResult
```